jwst/resample/resample_utils.py

```diff
@@ -135,6 +134,7 @@ def reproject(wcs1, wcs2):
     Parameters
     ----------
     wcs1, wcs2 : `~astropy.wcs.WCS` or `~gwcs.wcs.WCS` or `~astropy.modeling.Model`
+        or `~astropy.wcs.wcsapi.SlicedLowLevelWCS`
```

Comment:
```suggestion
    wcs1, wcs2 : `~astropy.wcs.WCS` or `~gwcs.wcs.WCS`
```

Reply:
I added a few words to the description to make it clearer that any WCS objects with the `pixel_to_world_values` and `world_to_pixel_values` attributes are acceptable.